modules/programs/freetube.nix

```diff
@@ -0,0 +1,53 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  inherit (pkgs.stdenv.hostPlatform) isLinux;
+  # FIXME-REDOX(Krey): Integrate
+  #inherit (pkgs.stdenv.hostPlatform) isRedox;
+  # FIXME-HURD(Krey): Integrate
+  #inherit (pkgs.stdenv.hostPlatform) isHurd;
+  # FIXME-FREEBSD(Krey): Integrate
+  #inherit (pkgs.stdenv.hostPlatform) isFreebsd;
+  inherit (pkgs.stdenv.hostPlatform) isCygwin;
+  inherit (pkgs.stdenv.hostPlatform) isDarwin;
+
+  cfg = config.programs.freetube;
+
+  # Config paths are on https://docs.freetubeapp.io/usage/data-location/
+  freetubeConfigPath =
+	if isLinux
+	  # DNM(Krey): Verify that $HOME works
+	  then "$HOME/.config/FreeTube"
+	else if isCygwin
+	  # DNM(Krey): Verify that `%APPDATA%` works
+	  then "%APPDATA%/FreeTube"
```

Comment:
We don't have a machine to test this in the CI and AFAIK we do not currently support this platform.
```suggestion
```

Reply:
GitHubCI offers Windows Machine to run tests on, i can try to integrate if if you want (would do a separate MR)